modules/ossm-config-openshift-monitoring-kiali.adoc

```diff
@@ -0,0 +1,75 @@
+// Module included in the following assemblies:
+//
+// * service-mesh-docs-main/observability/ossm-kiali-assembly.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="ossm-config-openshift-monitoring-kiali_{context}"]
+= Configuring OpenShift Monitoring with Kiali
+:context: ossm-config-openshift-monitoring-kiali
+
+// TP 1 content.
+// Possible file name may change
+// Possible assembly file may change
+// Assemblies, topic map info needs to be worked out still for 3.0.
+// Possible content will change.
+
+The following steps show how to integrate the {KialiProduct} with `user-workload monitoring`.
+
+.Prerequisites
+
+* {SMProductName} is installed.
+
+* `user-workload monitoring` is enabled. See "Enabling user-defined projects".
+
+* {KialiProduct} 1.89 is installed.
+
+.Procedure
+
+. Create a `ClusterRoleBinding` resource for Kiali:
++
+.Example `ClusterRoleBinding` configuration
+[source,yaml]
+----
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: kiali-monitoring-rbac
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: cluster-monitoring-view
+subjects:
+- kind: ServiceAccount
+  name: kiali-service-account
+  namespace: istio-system
+----
+
+. Create a Kiali resource and point it to your Istio instance:
```

Comment:
Not super critical but if this step is telling you to point the resource to the Istio instance, is there a way to signify how or what part of this step achieves that? For instance, if that part of the step is done by specifying the same namespace as the Istio instance, could you add a callout next to the `namespace` parameter in the example and add relevant text in the callout description?